src/components/Pagination.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Link from 'gatsby-link'
+
+const NavLink = ({ url, text }) => (
+  <Link
+    to={url}
+    activeClassName="font-bold"
+    className={`${
+      !url ? 'text-grey-lighter pointer-events-none' : ''
+    } no-underline mx-4`}
+  >
+    {text}
+  </Link>
+)
+
+export default function Pagination({
+  currentPage,
+  numberOfPages,
+  nextPagePath,
+  previousPagePath
+}) {
+  return (
+    <div className="flex justify-center items-center my-2">
+      <NavLink url={previousPagePath} text="‹‹" />
+      {Array.from({ length: numberOfPages }).map((_, i) => {
```

Comment:
This could be moved up into a const before `return` that uses [`useMemo`](https://reactjs.org/docs/hooks-reference.html#usememo) and has `numberOfPages` as a dependency. That way you wouldn't be recalculating the Array on every render.